lib/src/segment_config.dart

```diff
@@ -1,22 +1,25 @@
+enum SegmentIntegrationItemType { amplitude, unknown }
```

Comment:
Whats the point of `unknown`?

Reply:
Can we migrate all the existing options into this enum?

Reply:
@danielgomezrico 
> Whats the point of `unknown`?

I use to generate the model using json_serializable. I think this will be useful in the future (may be we gonna add .getSupportedList()). when our mapping between darts and native (ios/android) doesn't match at all. So, we can simply ignore it for now or just remove it. wdyt ?

> Can we migrate all the existing options into this enum?

What options do you want to migrate? If that means all segment integrations are available (amplitude, mixpanel, facebook, etc). Yes, we can add them easily when we add new integrations on the native side.